### ISSUE TITLE
Add the OTF:s?

### ISSUE THREAD

**Issue by August-Alm** [YuanshengZhao/Garamond-Math#2]
Could you please add the otf's? I have experience with LuaLaTeX but not with FontForge and can't figure out how to generate the files necessary from the sfd's. Either add the generated otf's or extend the Readme with some pointers on how to go about using your work.

**Comment by JakobWerner** [comment]
I agree. I just found the project and I would really like to test the font.
